### PR TITLE
Update 'Getting started'-step

### PR DESCRIPTION
**Pull request by c0ffeeca7**
- Addresses issue: https://github.com/home-assistant/yellow.home-assistant.io/issues/36
- Replace screenshot
- Add QR codes to link to Google Play Store and iOS App Store

### CODE REVIEW

**Comment by agners** [comment]
The QR codes look great! Can we squeeze the two apps in one step? Or even all three options (web/android/iOS). Since those three things are in a way different options of that last step

**Comment by c0ffeeca7** [comment]
> The QR codes look great! Can we squeeze the two apps in one step? Or even all three options (web/android/iOS). Since those three things are in a way different options of that last step

Thank you for the hint, @agners! I wanted to have it in one step, too, but I couldn't figure out how to change the template to allow multiple images in one step. That's why I used 3 steps instead. I used a workaround now.

File: src/procedures/procedures.11tydata.yaml [hunk 1]
```diff
@@ -6,23 +6,24 @@ procedureStandard:
...
-          <li>Once your Home&nbsp;Assistant Yellow has finished setting up, the onboarding will guide you through setup on your phone or computer</li>
-          <li>For further information, refer to the instructions provided under <a href="https://www.home-assistant.io/getting-started/onboarding/" target="_blank">Onboarding</a></li>
+          <li class="info">Initial startup may take a while, depending on your internet connection.</li>
+          <li><b>Desktop</b>: Visit <a href="http://homeassistant.local:8123" target="_blank">http://homeassistant.local:8123</a> in your browser to access the web frontend.</li>
+          <li>The web frontend guides you through the initial onboarding.</li>
+          <li class="info">For further information, refer to the instructions provided under <a href="https://www.home-assistant.io/getting-started/onboarding/" target="_blank">https://www.home-assistant.io/getting-started/onboarding/</a>.</li></br>
+          <li><b>Installing the app</b></br>To access Home Assistant from your mobile device, use the QR code to locate the app in the app store.</li>
```
Comment (agners) [inline src/procedures/procedures.11tydata.yaml:26]: Both work for initial onboarding, mobile or desktop. Can you move this point one up, and align it with Desktop (start with <b>Mobile</b>)

Comment (c0ffeeca7) [reply src/procedures/procedures.11tydata.yaml]: Good point. Thank you, @agners .

**Review by agners — APPROVED** [review]

**Pull request merged**